### PR TITLE
Fix duplicate context record generation when using imported proto file in gRPC tool

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
$Subject
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1499

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: grpc-ballerina/tests/grpc_tooling_common_test.bal [hunk 1]
```diff
@@ -105,6 +105,18 @@ function testHelloWorldWithDuplicateInputOutput() {
     }
 }
 
+@test:Config {enable:true}
+function testHelloWorldChild() {
+    var result1 = assertGeneratedDataTypeSources("data-types", "child.proto", "child_pb.bal", "tool_test_data_type_14");
+    if (result1 is error) {
```
Comment (BuddhiWathsala) [inline grpc-ballerina/tests/grpc_tooling_common_test.bal:111]: Shall we remove the unnecessary parathesis here?

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Review by dilanSachi — APPROVED** [review]
LGTM

File: grpc-ballerina/tests/grpc_tooling_common_test.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply grpc-ballerina/tests/grpc_tooling_common_test.bal]: Removed in 94d47a1

**Pull request merged**